pcli/src/main.rs

```diff
@@ -219,6 +237,29 @@ async fn main() -> Result<()> {
                 tracing::info!("got fragment: {:?}", block);
             }
         }
+        Command::AssetLookup { asset_id } => {
+            // xx don't hardcode, need to pass multiple ports on the command line
```

Comment:
This comment can be deleted here in `Command::AssetLookup` and below in `Command::AssetList` (leftover from a prior version before we were passing in the port of the wallet service on the command line)